packages/grafana-ui/src/components/Forms/Button.tsx

```diff
@@ -142,17 +145,38 @@ export const getButtonStyles = stylesFactory(({ theme, size, variant, withIcon }
   };
 });
 
-export const Button: FC<Omit<AbstractButtonProps, 'theme'>> = ({
-  renderAs,
-  size = 'md',
-  variant = 'primary',
-  className,
-  icon,
-  children,
-  ...otherProps
-}) => {
-  const theme = useTheme();
-  const buttonStyles = getButtonStyles({ theme, size, variant, withIcon: !!icon });
-
-  return renderButton(theme, buttonStyles, renderAs, children, size, variant, icon, className, otherProps);
+// There are but different from the standard button
```

Comment:
I don't get the meaning of this comment 🤔

Reply:
Fixed, written that one a bit too fast :D